test/multiverse/suites/roda/roda_instrumentation_test.rb

```diff
@@ -128,7 +128,10 @@ def test_rack_request_params_error
   end
 
   def assert_logged(expected)
-    found = NewRelic::Agent.logger.messages.flatten.any? { |m| m.match?(expected) }
+    # Example logger array:
+    # [[:debug, ["NoMethodError : undefined method `path' for \
+    # {}:Hash - Error encountered trying to identify Roda transaction name"], nil]]
```

Comment:
Long term we could possibly benefit from some logger helpers in the `test/` dir, but for now it really helps to have the example of the array so it is clear what `[1][0]` does.